conjur/cli.py

```diff
@@ -19,6 +19,7 @@
 # Internals
 from conjur.argparse_wrapper import ArgparseWrapper
 from conjur.client import Client
+from conjur.login import NetrcData, LoginLogic, LoginController, CredentialsFromFile
```

Comment:
TODO: Might need to change the name Login<xyz> because this logic ideally covers both Login **and** Logout